AvansDevOps/ForumComposite/ForumComponent.cs

```diff
@@ -8,6 +8,7 @@ namespace AvansDevOps.ForumComposite
 {
     public interface ForumComponent
     {
-        string GetMessage();
+        List<ForumComponent> GetComponents();
```

Comment:
Is dit niet iets dat alleen de composite component mag ?

Reply:
Je moet op een manier de composites op kunnen halen en bij diepere threads een message toe kunnen voegen dus dat doe ik nu op deze manier